packages/expo-cli/src/commands/build-native/__tests__/credentials-test.ts

```diff
@@ -83,10 +103,93 @@ describe('ensureCredentialsAsync', () => {
       (prompts as any).mockImplementation(() => {
         return { select: 'remote' };
       });
-      const src = await ensureCredentialsAsync(provider, Workflow.Generic, CredentialsSource.AUTO);
+      const src = await ensureCredentialsAsync(
+        provider,
+        Workflow.Generic,
+        CredentialsSource.AUTO,
+        false
+      );
       expect(prompts).toHaveBeenCalledTimes(1);
       expect(src).toBe('remote');
     });
+    it('should ask when local and remote are not the same (non interactive)', async () => {
+      const provider = crateMockCredentialsProvider({
```

Comment:
```suggestion
      const provider = createMockCredentialsProvider({
```
It'd be great if you could fix this typo in other places as well. Thanks!